action.yml

```diff
@@ -7,11 +7,11 @@ inputs:
     required: false
     default: master
   deploy_package_version:
-    default: "1.0.1.315"
```

Comment:
Is there a reason we're reverting the package back to 301?

Reply:
Weird. Corrected